galata/test/jupyterlab/notebook-mobile.test.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { test } from '@jupyterlab/galata';
+
+import { expect } from '@playwright/test';
+
+test.describe('Notebook Layout on Mobile', () => {
+  test('Execute code cell', async ({ page }) => {
+    await page.notebook.createNew();
+    await page.notebook.setCell(0, 'code', 'print("hello")');
+    await page.notebook.addCell('code', '2 * 3');
+    await page.notebook.runCellByCell();
+    await page.sidebar.close('left');
+    await page.setViewportSize({ width: 512, height: 768 });
```

Comment:
Not sure why, but on this branch it seems like the `setViewportSize` must be called here instead of `test.use({ viewport: { width: 512, height: 768 } })` as in the original PR.

Based on the failure noticed in: https://github.com/jupyterlab/jupyterlab/pull/11778#issuecomment-1004676272

Reply:
Could it be related to the version of playwright (in 3.2.x the lock file is on 1.16.2 when it is using 1.17.1 on master)?
Although I did not found any mention in the release notes.

Reply:
It could be yes. According to the video in https://github.com/jupyterlab/jupyterlab/pull/11778#issuecomment-1004676272, there seemed to be some troubles changing the cell type on a narrower viewport.